react/features/base/conference/functions.ts

```diff
@@ -53,7 +52,7 @@ export const getIsConferenceJoined = (state: Object) => Boolean(getConferenceSta
  * @returns {Promise}
  */
 export function _addLocalTracksToConference(
-        conference: { addTrack: Function, getLocalTracks: Function },
+        conference: { addTrack: Function; getLocalTracks: Function; },
```

Comment:
should be define some IJitsiConference, while we get LJM in order? With the stuff we use / need that is, it need not be complete...